doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -88,6 +92,13 @@ TLS 1.3 cipher suites) B<NID_auth_any> is returned. Examples (not comprehensive)
 SSL_CIPHER_is_aead() returns 1 if the cipher B<c> is AEAD (e.g. GCM or
 ChaCha20/Poly1305), and 0 if it is not AEAD.
 
+SSL_CIPHER_find() returns a B<SSL_CIPHER> structure which has the cipher ID B<ptr>.
```

Comment:
I'm not entirely comfortable saying "has the cipher ID ptr", as the natural interpration is that the pointer value is what is important, as opposed to the value pointed at, which is the actual case.  So, some language like "indicated by" or "stored in" or similar would be an improvement.